### PR TITLE
Feat: use aind-ng-link for neuroglancer link creation

### PR DESCRIPTION
**Pull request by carshadi**

### CODE REVIEW

File: post_install.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-python -m pip install "git+https://github.com/AllenNeuralDynamics/aind-ng-link@feat-zarr-checker#egg=aind-ng-link" --no-cache-dir
+python -m pip install "git+https://github.com/AllenNeuralDynamics/aind-ng-link" --no-cache-dir
```
Comment (jtyoung84) [inline post_install.sh:3]: Is this published to pypi now? We can probably just add it to the pyproject.toml file

Comment (carshadi) [reply post_install.sh:3]: I think @dyf had published an older version at some point https://pypi.org/project/aind-ng-link/1.0.6/
Not sure if the latest version is up.

Comment (jwong-nd) [reply post_install.sh:3]: Ah no auto-publishing, let me publish the latest version

Comment (jwong-nd) [reply post_install.sh:3]: @carshadi Try this out: 
https://pypi.org/project/aind-ng-link/1.0.15/

Comment (carshadi) [reply post_install.sh]: works 👍

**Comment by carshadi** [comment]
@jtyoung84 updated

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**